### PR TITLE
Update github files

### PR DESCRIPTION
**Pull request by camilamacedo86**

### CODE REVIEW

**Review by aliok — COMMENTED** [review]
@camilamacedo86 thanks for this update.

Left a very small comment feel free to merge afterwards

**Comment by camilamacedo86** [comment]
Hi @aliok,

Thank you for your help here. I could not find the small comment could you check this? What do you would like to change here? Also, could you approve the PR in this way I will do this change and merge it as soon as possible?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Motivation
+Add references to relevant tickets or a short description about what motivated you do it. (E.g JIRA: https://issues.jboss.org/browse/AEROGEAR-{} AND/OR ISSUE: https://github.com/aerogear/standards/issues/{})
```
Comment (aliok) [inline .github/PULL_REQUEST_TEMPLATE.md]: @camilamacedo86 
Can we change the GH issues url with https://github.com/aerogear/ios-showcase-template/{}

**Review by aliok — APPROVED** [review]

**Comment by aliok** [comment]
@camilamacedo86 somehow my comment was lost. anyway, readded it and approved the PR.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (camilamacedo86) [reply .github/PULL_REQUEST_TEMPLATE.md]: Sure 👍

**Pull request merged**